### PR TITLE
Rewrite commands to use command executor objects

### PR DESCRIPTION
**Pull request by kingbirdy**
- New executor class for each command, extends new abstract PKCommand

- Rewrote Commands to use the new objects

- Update references to old Commands class

### CODE REVIEW

File: src/com/projectkorra/projectkorra/ability/AbilityModuleManager.java [hunk 1]
```diff
@@ -353,12 +354,14 @@ private void fill() {
 				descriptions.put(ab.getName(), ab.getDescription());
 				authors.put(ab.getName(), ab.getAuthor());
 			}
-			catch (AbstractMethodError /* pre 1.6 BETA 8 */| NoSuchMethodError /*
+			catch (AbstractMethodError /* pre 1.6 BETA 8 */ | NoSuchMethodError /*
 																				 * pre
-																				 * 1.7
+																				 * 1
```
Comment (jacklin213) [inline src/com/projectkorra/projectkorra/ability/AbilityModuleManager.java]: Uhhh what happened here

File: src/com/projectkorra/projectkorra/commands/AddCommand.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			if (!(sender instanceof Player) || !((Player) sender).equals(target)) {
+				if (element.charAt(0) == 'w' || element.charAt(0) == 'f' || element.charAt(0) == 'c') {
+					sender.sendMessage(ChatColor.DARK_AQUA + target.getName() + color + " is also a " + Character.toString(element.charAt(0)).toUpperCase() + element.substring(1) + "bender.");
+				} else if (element.charAt(0) == 'e' || element.charAt(0) == 'a') {
+					sender.sendMessage(ChatColor.DARK_AQUA + target.getName() + color + " is also an " + Character.toString(element.charAt(0)).toUpperCase() + element.substring(1) + "bender.");
+				}
+			}
+			Bukkit.getServer().getPluginManager().callEvent(new PlayerChangeElementEvent(sender, target, type, Result.ADD));
+			return;
+		} else
```
Comment (jacklin213) [inline src/com/projectkorra/projectkorra/commands/AddCommand.java]: {} need to be fixed here

File: src/com/projectkorra/projectkorra/commands/ClearCommand.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+					sender.sendMessage(ChatColor.RED + "The slot must be an integer between 1 and 9.");
+				}
+				if (bPlayer.getAbilities().get(slot) != null) {
+					bPlayer.getAbilities().remove(slot);
+					GeneralMethods.saveAbility(bPlayer, slot, null);
+					sender.sendMessage("You have cleared slot #" + slot);
+				} else {
+					sender.sendMessage("That slot was already empty.");
+				}
+			}
```
Comment (jacklin213) [inline src/com/projectkorra/projectkorra/commands/ClearCommand.java]: Collapse Line 55 & 56

Comment (kingbirdy) [inline src/com/projectkorra/projectkorra/commands/ClearCommand.java]: Make the formatter do it automatically :P

Comment (jacklin213) [inline src/com/projectkorra/projectkorra/commands/ClearCommand.java]: Does it not?

Comment (kingbirdy) [inline src/com/projectkorra/projectkorra/commands/ClearCommand.java]: no

**Pull request merged**